core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -117,6 +117,8 @@ class LogManager(logDirs: Seq[File],
   }
 
   private val dirLocks = lockLogDirs(liveLogDirs)
+  private val dirIds = directoryIds(liveLogDirs)
+  val directoryIds: Set[Uuid] = dirIds.values.toSet
```

Comment:
Seems this is only used in the test, so can we do this instead?
```
  // visible for testing
  private[log] val directoryIds: Set[Uuid] = dirIds.values.toSet
```

Reply:
Thanks. Updated.